### DISCUSSION TITLE
Backup and Media Data Storage on an External NAS

### DISCUSSION THREAD

**Discussion by Buhli87** [photoprism/photoprism#4654] [Q&A]
Hello everyone,

I am quite new to the world of Proxmox and PhotoPrism.

According to https://tteck.github.io/Proxmox/, I installed PhotoPrism and it worked. 

In the container itself, I added a mount point under "Resources": /mnt/pve/nas/Photos,mp=/Photos.
![image](https://github.com/user-attachments/assets/82fcf7ec-fafe-49f5-b473-6146210bfd9a)

I adjusted the /opt/photoprism/config/.env file as follows:

PHOTOPRISM_AUTH_MODE='password' 
PHOTOPRISM_ADMIN_PASSWORD='***************' 
PHOTOPRISM_HTTP_HOST='0.0.0.0' 
PHOTOPRISM_HTTP_PORT='2342' 
PHOTOPRISM_SITE_CAPTION='https://tteck.github.io/Proxmox/' 
PHOTOPRISM_STORAGE_PATH='/opt/photoprism/storage' 
**PHOTOPRISM_ORIGINALS_PATH='/Photos/Originals' 
PHOTOPRISM_IMPORT_PATH='/Photos/Import' 
PHOTOPRISM_CACHE_PATH='/Photos/Thumbnails' 
PHOTOPRISM_BACKUP_PATH='/Photos/Backup'**

The import works, and I can see the images. However, when I run the command: **photoprism backu -i test**, I get the error: "**config: invalid originals path, check configuration and permission**."

Can someone tell me what I did wrong? Or is my /.env file okay as it is? Or should it be done differently?

Thank you very much,

Best regards, Buhli

**Comment by omerdduran** [comment]
Hi. Please use the “photoprism config” command to see a list of all configuration options and their current values and check if they match your settings in the “.env” file. If it doesn't match, you can also set the config as in this document 

https://docs.photoprism.app/getting-started/config-files/#optionsyml

**Comment by Buhli87** [comment]
@omerdduran, thank you very much for your response.

I checked this "photoprism config" file and I'm a bit confused for the result. 

For example. In this file the "originals-path" is empty. But I have set this value in the .env file. The same goes for the import-path, cache-path, and backup-path. Some are empty and some has other values.

![image](https://github.com/user-attachments/assets/054fee3c-a078-454f-b833-0ebc5d196cd4)

![image](https://github.com/user-attachments/assets/05e71858-0b82-4114-9138-bdf631953b17)

What would be the solution here? Should I also include / correct the values in this "photoprism config" file?

Thank you very much.

Best regards,
Buhli

**Reply by omerdduran** [reply]
There seems to be a problem reading the env file. Maybe you need to use “ instead of '. If that doesn't work, you can do it with options.yml as in this example https://docs.photoprism.app/getting-started/config-files/#optionsyml 

Create `options.yml` in your config path (typically `/opt/photoprism/storage/config/options.yml`)

Example configuration:
```yaml
OriginalsPath: "/photoprism/originals"
ImportPath: "/photoprism/import"
StoragePath: "/photoprism/storage" 
CachePath: "/photoprism/cache"
BackupPath: "/photoprism/backup"
```

Restart PhotoPrism for changes to take effect

**Comment by Buhli87** [comment]
@omerdduran thank you very much

> There seems to be a problem reading the env file. Maybe you need to use “ instead of '

You really think there is a problem to read the env file? Even if the import works and I see the pictures?

Thanks and best regards
Buhli

**Comment by Buhli87** [comment]
@omerdduran  I have tested it, unfortunately without success

1. I replaced all ' with “
2. I have created an `options.yml` file with the following value: **OriginalsPath: "/Photos/Originals"**, removed the originials path value from the `env` file and rebooted the conatainer. After the reboot I checked the `photoprism config` but there the orginals-path was still empty.
3. Just for fun I replaced in the `options.yml` file the value from OriginalsPath: **"/Photos/Originals** to OriginalsPath: **"/photoprism/originals"** and after the reboot, I saw this path in the `photoprism config`
4. I created a new mount point (mp=/photoprism/data/Originals), changed the path in `options.yml` file, rebooted the containter and checked the `photoprism config` again. But there the path was empty.

At the moment I don't understand the whole behavior. Why does it sometimes have no or wrong paths, but the import works and the images are displayed?

Thanks and best regards
Buhli
